_includes/docs/extending/installing.md

```diff
@@ -0,0 +1,15 @@
+You can install extensions via the UI or the command line.
+
+### UI
+Modules can be installed by going to [http://yoursite.tld/admin/modules](), or by clicking on `Extend` in the menu at the top.  
+Here you can search for them, by entering the name in the filter box at the top.  
+To actually install a module, select the checkbox next to it, scroll to the bottom and click `Install`. You might be warned, that another module needs to be enabled, because it is required for the module of your interest. By clicking on `continue`, Drupal will take care of that.
+
+Themes can be installed by going to [http://yoursite.tld/admin/appearance](), or by clicking on `Appearance` in the menu at the top.  
+Here you can scroll to the theme, you would like to install and click on `Install`, to install it, or click on `Install and set as default`, to use the theme for your site, after installing it.  
```

Comment:
Add comma: "like to install, and click"
Delete commas: "'Install' to install it, or on 'Install and...' to use the theme for your site after installing it"